students/15th/chaehoonpark/js/login.js

```diff
@@ -0,0 +1,52 @@
+// 로그인 페이지 js 파일 예정
+
+// 로그인 버튼 활성화 컬러 : #0095F6
+const thisIsButton = document.getElementsByTagName("button")[0];
+const thisIsPassword = document.getElementById("Password");
+const thisIsId = document.querySelector("#ID");
+const thisIsInputBox = document.querySelector(".input_and_button_box");
+
+// thisIsButton.addEventListener('keyup',function(){
+//     let thisIsId = document.getElementById('ID');
+//     let thisIsPassword = document.getElementById('Password');
+
+//     if((thisIsId.length === 1) && (thisIsPassword.length === 6)){
+//         thisIsButton.style.color = "#0095F6";
+// }});
+// 이 코드가 안 됐던 이유는 button 에다가 event 를 걸었기 때문이다. event 가 일어나는 곳은 ID, PASSWORD인데
+
+thisIsInputBox.addEventListener("keyup", function () {
+  // if(event.target.value){
+  //     thisIsButton.style.backgroundColor = "#0095F6";
+  // }else{
+  //     thisIsButton.style.backgroundColor = "#B2DFFC";
+  // }
+  const thisIsIdValue = thisIsId.value;
+  const thisIsPasswordValue = thisIsPassword.value;
+//   if (thisIsIdValue && thisIsPasswordValue) {
+//     thisIsButton.style.backgroundColor = "#0095F6";
+//   } else {
+//     thisIsButton.style.backgroundColor = "#B2DFFC";
+//   }
+
+  // if(thisIsIdValue.length !== 0 && thisIsPasswordValue.length !== 0){
+  //     thisIsButton.style.backgroundColor = "#0095F6";
+  // }else{
+  //     thisIsButton.style.backgroundColor = "#B2DFFC";
+  // }
+
+  !!(thisIsIdValue && thisIsPasswordValue) ? (thisIsButton.style.backgroundColor = "#0095F6")
+  : (thisIsButton.style.backgroundColor = "#B2DFFC");
```

Comment:
삼항연산자 활용 잘 해주셨네요! 굿굿! 
아래와 같이 사용하시면 더 가독성에 좋습니다 ;) 
```suggestion
  thisIsButton.style.backgroundColor =
    thisIsIdValue && thisIsPasswordValue ? "#0095F6" : "#B2DFFC";
```

Reply:
멘토님 궁금한 게 있습니다! 

!!(thisIsIdValue && thisIsPasswordValue) 이 부분에서 처음에는 (thisIsIdValue && thisIsPasswordValue) 이와 같이 코딩을 했었는 데 생각한대로 동작이 되지 않았습니다. console.log(thisthisIsIdValue) 하고 (thisIsPasswordValue) 를 해보니thisthisIsId 와 thisIsPassword 의 value 는 찍히던데, !! 를 쓰지 않으면 안되는 건가요?

Reply:
채훈님, !!를 사용하지 않아도 동작이 되는 것 같은데...ㅎㅎ 어떤 문제였을까요?! 다시 한번 해보시겠어요?!

Reply:
다시 해보니까 되네욥.... 반성하겠습니다... ㅠ